visivo_project.yml

```diff
@@ -22,6 +36,76 @@ models:
         action = 'submitted'
 
 traces:
+  - name: Average Cycle Time Indicator
+    model: ref(closed github pull requests)
+    target_name: "{{env_var('GITHUB_TARGET')}}"
+    columns: 
+      cycle_time: avg(cycle_time_hrs)
+    props:
+      type: indicator
+      value: column(cycle_time)[0]
+      title: 
+        text: Average Cycle Hours
+  - name: Pull Request Reviews Indicator
+    model: ref(submitted github pull requests reviews)
+    target_name: "{{env_var('GITHUB_TARGET')}}"
+    columns: 
+      pr_reviews: count(*)
+    props:
+      type: indicator
+      value: column(pr_reviews)[0]
+      title: 
+        text: PR Reviews All Time
+  - name: Pull Request Opens Indicator
+    model: ref(opened github pull requests)
+    target_name: "{{env_var('GITHUB_TARGET')}}"
+    columns: 
+      pr_opens: count(*)
+    props:
+      type: indicator
+      value: column(pr_opens)[0]
+      title: 
+        text: PR Opens All Time
+  - name: Average Cycle Time By Repository This Month
```

Comment:
Seeing this list, I am thinking that a great command to the cli would be `visivo list trace` and it would give you a list of available traces.

Reply:
Agreed. I think there's something to be said for auto completion as well. Made a card for it: https://linear.app/visivo/issue/VIS-262/auto-complete-within-refs